app/serializers/project_serializer.rb

```diff
@@ -6,12 +6,12 @@ class ProjectSerializer
              :subjects_count, :created_at, :updated_at, :available_languages,
              :title, :description, :guide, :team_members, :science_case,
              :introduction, :avatar, :background_image, :private, :faq, :result,
-             :education_content, :retired_subjects_count, :avatar, :background_image,
-             :configuration, :beta, :approved
+             :education_content, :retired_subjects_count, :configuration,
+             :beta, :approved, :live
 
   can_include :workflows, :subject_sets, :owners, :project_contents,
               :project_roles
-  can_filter_by :display_name, :beta, :approved
+  can_filter_by :display_name, :beta, :approved, :live
```

Comment:
Added it in in case there's a use for it in the future, i.e. find all the projects that are in the live state, I should probably add and index on it if that's the case.